client/src/styles/login.css

```
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  /* user-select: none; */
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+html,
+body {
+  height: 100%;
+}
+/* body{
+  display: grid;
+  place-items: center;
+  background: #dde1e7;
+  text-align: center;
+  align-items: center;
+} */
+.login-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
```

```suggestion
    height: calc(100vh - 43px);
```

This will stop the scrollbar from appearing on the login page

---

Corrected.